cli/scaletest.go

```diff
@@ -947,6 +901,167 @@ func (r *RootCmd) scaletestCreateWorkspaces() *clibase.Cmd {
 	return cmd
 }
 
+func (r *RootCmd) scaletestTrafficGen() *clibase.Cmd {
+	var (
+		duration        time.Duration
+		bps             int64
+		client          = new(codersdk.Client)
+		tracingFlags    = &scaletestTracingFlags{}
+		strategy        = &scaletestStrategyFlags{}
+		cleanupStrategy = &scaletestStrategyFlags{cleanup: true}
+		output          = &scaletestOutputFlags{}
+	)
+
+	cmd := &clibase.Cmd{
+		Use:    "trafficgen",
```

Comment:
workspace-traffic?

"traffic" could also be to the Coder API, e.g. dashboard stuff.

Reply:
Renamed to `workspace-traffic`. I can imagine `dashboard-traffic` and `ssh-traffic` down the line.